includes/class-wc-cart.php

```diff
@@ -1700,7 +1700,7 @@ public function add_discount( $coupon_code ) {
 			}
 
 			// If its individual use then remove other coupons
-			if ( $the_coupon->individual_use == 'yes' ) {
+			if ( true === $the_coupon->get_is_individual_use() ) {
```

Comment:
Is this best practice to include the comparison? Asking because get_is_individual_use casts to bool so seems unnecessary.